integration-test/473-landuse-tier.py

```diff
@@ -13,8 +13,7 @@
 # area 30089300
 assert_has_feature(
     8, 56, 115, 'landuse',
-    { 'kind': 'national_park', 'id': -921675, 'tier': 1,
-      'min_zoom': 8 })
+    { 'kind': 'national_park', 'tier': 1, 'min_zoom': 8 })
 assert_no_matching_feature(
     7, 28, 57, 'landuse',
     { 'kind': 'national_park', 'id': -921675 })
```

Comment:
It passed without it, so I didn't mess with it, but happy to change. Would it be better to just bump the zoom level up one instead? Not quite exactly the same test then (testing the min zoom rather than when the tile actually shows up), but probably equivalent enough?